.github/workflows/pages.yml

```diff
@@ -0,0 +1,27 @@
+on: 
+  push:
+    branches: 
+    - master
+jobs:
+  run:
+    runs-on: ubuntu-18.04
+    steps:
+    # - uses: docker://pandoc/latex:2.9
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # (x64 or x86)
+    - run: |
+        sudo apt install pandoc
+        # add dependencies based on the conf.py
+        pip install sphinx myst-parser sphinx-copybutton sphinx-markdown-tables
```

Comment:
`pip install -r `requirements.txt`? shorter and probably more future-proof.

Reply:
this makes sense. where would you like the requirements to go?

Reply:
They're already present in the repo root. With version pinning for Sphinx (safer), and `sphinx-material` which is missing here. I'll push a fix.